docs/order.rst

```diff
@@ -0,0 +1,142 @@
+.. _order:
+
+Ordering
+========
+
+Ordering is very similar to filtering. It is provided via method ``_order_queryset`` that you can override to change
+its default implementation. Resource class ``BaseModelResource`` again uses manager to simplify custom modifications::
+
+    class BaseModelResource(DefaultRESTModelResource, BaseObjectResource):
+        order_manager = DefaultModelOrderManager()
+
+        def _order_queryset(self, qs):
+            if self.order_manager:
+                return self.order_manager.order(self, qs, self.request)
+            else:
+                return qs
+
+Method ``_order_queryset`` uses manager to order/sort output data. You can change manager with ``order_manager``
+attribute. If you set ``order_manager`` to ``None``. Filtering will be disabled. Default order manager is
+``DefaultModelOrderManager``.
+
+Order manager
+-------------
+
+Order manager purposes are:
+
+* parse input data that contains information about ordering and split this data to pairs <order identifier, order direction>
+* check if concrete identifier is allowed to order
+* convert identifiers to django ordering identifiers
+* finally apply ordering on django QuerySet
+
+Allowed identifiers defines ``order_fields`` or ``extra_order_fields`` that can be defined on model or resource (or both).
+
+DefaultModelOrderManager
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is only one pre-implemented Pyston ordering manager. Input data is parsed from querystring named ``order`` or
```

Comment:
It is the only pre-implemented Pyston ordering manager. Input data is parsed from querystring named ``order`` or HTTP header named ``X-Order``. Both have the same format. Order terms are split with a ``,`` char, each term consits of a direction char and an identifier. Missing direction char means ascending and char ``-`` means descending order. Identifiers are very similar to django identifiers where relations are joined with a string ``__``. Example: